packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -4,6 +4,36 @@
 
 import 'package:meta/meta.dart';
 
+const String _currentDirectoryScript = r'''
```

Comment:
A comment explaining the dark magic below could help a lot.

Reply:
Done